cfgov/scripts/fix_links_in_tableblocks.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import get_object_or_404
+from bs4 import BeautifulSoup
+
+from wagtail.wagtaildocs.models import get_document_model
+from wagtail.wagtailcore.models import Page
+
+from v1.models.base import CFGOVPage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+def get_tableblocks(page):
+    """ Get all the TableBlocks for a given page.
+    TableBlocks can be stored either directly in page's content,
+    or in a FullWidthText item. So we must check both.
+    """
+    try:
+        stream_data = page.specific.content.stream_data
+    except:
+        return []
+    tableblocks = list(filter(lambda item: item['type'] == 'table_block', stream_data))
+    full_width_text_items = list(filter(lambda item: item['type'] == 'full_width_text', stream_data))
+    for item in full_width_text_items:
+        sub_items = item['value']
+        for sub_item in list(filter(lambda i: i['type'] == 'table_block', sub_items)):
+            tableblocks.append(sub_item)
+    return tableblocks
+
+
+def convert_links(links):
+    """ Adds a href to the link with the reative path if a
+    document ID or page ID is stored
+    """
+    updated = False
+    for link in links:
+        linktype = link.get('linktype')
+        object_id = link.get('id')
+        if link.get('href'):
+            continue  # Don't update it if the href is already set
+        if linktype == 'document':
+            doc = get_object_or_404(get_document_model(), id=object_id)
+            filepath = '/' + doc.file.name
+        elif linktype == 'page':
+            page = Page.objects.get(id=object_id)
+            filepath = page.relative_url(current_site=page.get_site())
+        else:
+            continue
+        updated = True
+        link['href'] = filepath
+    return updated
+
+
+def run():
+    for page in CFGOVPage.objects.all():
+        changes = False
+        tableblocks = get_tableblocks(page)
+        for tableblock in tableblocks:
+            rows = tableblock['value']['data']
+            for row in rows:
+                for idx, item in enumerate(row):
+                    soup = BeautifulSoup(item, 'html.parser')
+                    links = soup.findAll('a')
+                    if links and convert_links(links):
+                        # Set the item to the modified HTML
+                        row[idx] = str(soup)
+                        changes = True
+        if changes:
+            publish_changes(page.specific)
```

Comment:
Why does this use `publish_changes` instead of `save_page`?

One risk of [`publish_changes`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/tests/wagtail_pages/helpers.py#L25) is that if a page is only in draft or shared mode, this code would publish it. I don't know if any of the existing `TableBlock` pages would be in that state, but it might be something we should keep an eye on in future.